### ISSUE TITLE
False positive on empty permissions object

### ISSUE THREAD

**Issue by mwarres** [rhysd/actionlint#178]
False positive on: `permissions: {}`

Provokes the following error message:

```tmp/lint/.github/workflows/test.yaml:45:18: "permissions" section should not be empty. please remove this section if it's unnecessary [syntax-check]                                                                                               |                                                                                                                    45 |     permissions: {}   ```

GitHub Actions allows this syntax.  See ScribeMD/docker-cache#81.

**Comment by shogo82148** [comment]
duplicated as https://github.com/rhysd/actionlint/issues/170

**Comment by rhysd** [comment]
This is already fixed on main branch. Closing.
